<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPosters</class>
 <widget class="QDialog" name="AddPosters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="1" colspan="2">
      <widget class="QSpinBox" name="countSeats">
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="countSeatsLabel">
       <property name="font">
        <font>
         <family>Mongolian Baiti</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>КОЛИЧЕСТВО МЕСТ</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QDateEdit" name="date"/>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="QLabel" name="labelError">
       <property name="font">
        <font>
         <family>Mongolian Baiti</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel { color: rgb(240, 0, 0); }</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="mainLabel">
       <property name="font">
        <font>
         <family>Mongolian Baiti</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ДОБАВЛЕНИЕ/РЕДАКТИРОВАНИЕ АФИШИ</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="namePerformanceLabel">
       <property name="font">
        <font>
         <family>Mongolian Baiti</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>НАЗВАНИЕ СПЕКТАКЛЯ</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="dateLabel">
       <property name="font">
        <font>
         <family>Mongolian Baiti</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ДАТА</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="namePerformance">
       <property name="font">
        <font>
         <family>Mongolian Baiti</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="timeLabel">
       <property name="font">
        <font>
         <family>Mongolian Baiti</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ВРЕМЯ</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QTimeEdit" name="time"/>
     </item>
     <item row="7" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCANCEL">
       <property name="text">
        <string>CANCEL</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOK</sender>
   <signal>clicked()</signal>
   <receiver>AddPosters</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCANCEL</sender>
   <signal>clicked()</signal>
   <receiver>AddPosters</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
